appsScript/CreateFormFields.js

```diff
@@ -0,0 +1,304 @@
+/**
+ * Function to create the licenses dropdown in the form, based upon the SPDX license list (Required)
+ */
+const CreateLicenseField = (form, name, required = false) => {
```

Comment:
Looks good. you have a bit of repeat code, i'm wondering if you could externalise some, with something like:

```
setTextItem(title, helpText, required){
textItem.setTitle(title)
textItem.setHelpText(helpText)
textItem.setRequired(required)
}
```
And the same for multiple choice items and listItems. Would that make it clearer?